apollo-ios-codegen/Sources/ApolloCodegenLib/Templates/EnumTemplate.swift

```diff
@@ -43,15 +47,19 @@ struct EnumTemplate: TemplateRenderer {
       \(if: shouldRenderDocumentation, "///")
       \(documentation: "**Deprecated**: \($0.escapedSwiftStringSpecialCharacters())")
       """ })
+    \(if: graphqlEnumValue.name.shouldRenderDocumentation, """
+      \(graphqlEnumValue.name.typeNameDocumentation)
+      """
+    )
```

Comment:
Could we just make `typeNameDocumentation` return `nil` if it shouldn't be rendered? The `TemplateString` will remove the line if an optional is appended and it is nil.

Then this could just be:
```suggestion
    \(graphqlEnumValue.name.typeNameDocumentation) 
```